### PR TITLE
Docs for eas apps tab

### PR DESCRIPTION
**Pull request by kmacgugan**
### :white_check_mark: Checklist

- [ ] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable

### CODE REVIEW

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+```bash
+curl -sSX POST "https://automate-url/apis/v0/retention/service_groups/disconnected_services/config" -d
+'{
+  "threshold": "15m"
+}'
+-H "api-token: $TOKEN"
+```
+
+### Periodic deletion
```
Comment (apriofrost) [inline components/automate-chef-io/content/docs/eas_applications.md]: Minor: replacing _deletion_ with _removal_ to match with the language used in the settings UI

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+In addition to the disconnection check, services that have been disconnected for a period of time are deleted. This helps to keep the dashboard populated with the most relevent information and saves on storage space over time. By default this deletion threshold is 7 days, but can be disabled if you wish to keep the record of services forever.
```
Comment (apriofrost) [inline components/automate-chef-io/content/docs/eas_applications.md]: Minor: update _deleted_  to _removed_; _deletion_ to _removal_.

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+To configure the threshold for deleting a disconnected service, you can do this via the UI or the API directly. The threshold is the time since it was marked for deletion. You can also disable deletion by setting running to false. To access the configuration settings in the UI, go to the settings tab, then Data Lifecycle on the sidebar. Under service groups you will find settings for Marking and Deleting disconnected services.
```
Comment (apriofrost) [inline components/automate-chef-io/content/docs/eas_applications.md]: Minor: update _deleting_ to _removing_; _deletion_ to _removal_.

**Comment by vjeffrey** [comment]
should we close https://github.com/chef/automate/issues/1962 since it is being done as part of this already?

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+## Requirements and Capacity Planning
+Based on the latest testing, the EAS Applications system can scale up to 15,000 services on a system with 4(v)CPUs, 16GB of RAM, and SSD disks.
+
+You must use habitat 0.83 or greater.
+
+At a larger scale, the subsystems responsible for processing applications data will consume significant CPU resources. If the Compliance and Infrastructure visibility features of Automate will also be used on this Chef Automate instance, the host system will need additional compute and memory capacity.
+
+## Configuring Chef Habitat to Send Events to Chef Automate
+In the Chef Automate UI, create an API token as described in the Chef Automate documentation. Save this token somewhere safe.
+Please see (https://www.habitat.sh/docs/habitat-cli/#hab-sup-run "habitat documentation") for more habitat options.
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: should this "please see.." sentence be down further, like under the section that says to launch with options?

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+--event-stream-environment=MY_ENV \
+--event-stream-site=MY_SITE \
+--event-stream-url=AUTOMATE_HOSTNAME:4222 \
+--event-stream-token=API_TOKEN
+```
+
+Replace MY_APP, MY_ENV, MY_SITE, AUTOMATE_HOSTNAME, and API_TOKEN with the appropriate values. MY_APP, MY_ENV, MY_SITE are user-defined and can be set to any string. MY_APP, MY_ENV are used to group services in the Chef Automate interface and are intended to be used to describe your applications and application lifecycle, as explained in the concepts section of this document. MY_SITE is intended to describe the physical (i.e., datacenter) or cloud-specific (i.e., AWS region) location where your services are deployed. These values are search enabled, allowing for filtering of services.
+
+## Lifecycle management
+A future release will enable you to manage the lifecycle of disconnected services via the UI. Currently the functionality is available via the command line and using the API directly. Using the command line will kick off a one time job to assess disconnection status or remove disconnected services. The API is used to manage the activity of periodic jobs to assess disconnection status and removal.
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: didn't the command line bits get removed?

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+Disconnected services are defined as services known to the EAS Applications backend storage that have not received a health check message for a user-defined period of time.
+
+### Periodic disconnection checks
+
+Automate assesses the status of the habitat services based on the receipt of a health check message. If a health check message has not been recieved for a configurable amount of time the service is marked as disconnected. The default amount of time is 5 minutes and this check cannot be disabled. Habitat send the health check messages every 30 seconds by default, this is also configurable using the `--health-check-interval` option with habitat supervisor. Please see (https://www.habitat.sh/docs/habitat-cli/#hab-sup-run "habitat documentation") for more information
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: "recieved" ->> "received"

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+To configure the threshold for marking a disconnected service, you can do this via the UI or the API directly. To access the configuration settings in the UI, go to the settings tab, then Data Lifecycle on the sidebar. Under service groups you will find settings for Marking and Removing disconnected services.
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: this doesn't exist yet, right?

Comment (vjeffrey) [reply components/automate-chef-io/content/docs/eas_applications.md]: i mean in the ui

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+Save the file with any name, then run chef-automate config patch FILENAME to apply the configuration change. You should see output similar to the following:
+
+Updating deployment configuration
+
+Applying deployment configuration
+  Started event-gateway
+
+### Configure automate for encrypted communication
+
+By default frontend TLS is enabled. If it was previously disable it can be re-enabled. On the host where Chef Automate is deployed, create a file with the following content:
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: "disable" --> "disabled"

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+```
+
+Save the file with any name, then run chef-automate config patch FILENAME to apply the configuration change. You should see output similar to the following:
+
+Updating deployment configuration
+
+Applying deployment configuration
+  Started event-gateway
+
+Once you have done that retrieve the automate frontend TLS key to use for configuring your habitat supervisor.
```
Comment (vjeffrey) [inline components/automate-chef-io/content/docs/eas_applications.md]: comma after "once you have done that" ?

File: components/automate-chef-io/content/docs/eas_applications.md (hunk 7 above)
Comment (kmacgugan) [reply components/automate-chef-io/content/docs/eas_applications.md]: It doesn't exist, but it should for GA so I added it.

**Comment by vjeffrey** [comment]
https://github.com/chef/automate/pull/2207

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+(https://automate.chef.io/docs/configuration/#load-balancer-certificate-and-private-key "here").
+
+### Configure habitat supervisor for encrypted communication
+
+There are three ways to share the automate certificate with the habitat supervisor to use when connecting to automate.
+
+1) Pass the certificate file as a parameter to the --event-stream-server-certificate flag.
+
+```
+HAB_FEAT_EVENT_STREAM=1 hab sup run \
```
Comment (danielsdeleo) [inline components/automate-chef-io/content/docs/eas_applications.md]: Do we know when the hab feature flag will no longer be needed?

File: components/automate-chef-io/content/docs/eas_applications.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+
+## Requirements and Capacity Planning
+Based on the latest testing, the EAS Applications system can scale up to 15,000 services on a system with 4(v)CPUs, 16GB of RAM, and SSD disks.
+
+You must use habitat 0.83 or greater.
```
[the remark this answers is not in the document]
Comment (kmacgugan) [reply components/automate-chef-io/content/docs/eas_applications.md]: Update: it is 0.89.47 onward

File: components/automate-chef-io/content/docs/eas_applications.md (hunk 10 above)
Comment (kmacgugan) [reply components/automate-chef-io/content/docs/eas_applications.md]: @davidMcneil is going to bring it up with the hab team.

**Comment by kagarmoe** [comment]
@chef/docs-team needs guidance from product and marketing around EAS. Our draft content is in #2207.

**Comment by kagarmoe** [comment]
Could I get eyes on https://github.com/chef/automate/pull/2207 before we merge it here?

**Comment by mjingle** [comment]
This PR is good to go and ready to merge when applicable.

**Pull request merged**